### PR TITLE
use cli directly instead of making new executor

### PR DESCRIPTION
**Pull request by wandmagic**
Simplify by not using a custom executor.
Commands were change to private scoping so they can't be imported directly anymore

### CODE REVIEW

**Review by aj-stein-gsa — COMMENTED** [review]
Are the test failures just content update issues? There seem to be a few `ExitStatus.RUNTIME_ERROR`s in the pipeline failures in the test step.

**Comment by wandmagic** [comment]
Yeah not sure why but it fails to write to the sarif file

**Comment by wandmagic** [comment]
converting to a draft until i can resolve the test failures

**Comment by wandmagic** [comment]
i found the issue, because i had ERROR level only logging, the execution crashed because it depends on LOG4J for reporting errors

File: src/main/kotlin/gov/nist/secauto/oscal/tools/server/OscalVerticle.kt [hunk 1]
```diff
@@ -437,7 +452,7 @@ class OscalVerticle : CoroutineVerticle() {
                 "driver": {
                   "name": "OSCAL Tool",
```
Comment (david-waltermire) [inline src/main/kotlin/gov/nist/secauto/oscal/tools/server/OscalVerticle.kt:453]:
```suggestion
                  "name": "OSCAL Server",
```

File: src/main/kotlin/gov/nist/secauto/oscal/tools/server/OscalVerticle.kt [hunk 2]
```diff
@@ -437,7 +452,7 @@ class OscalVerticle : CoroutineVerticle() {
...
                   "informationUri": "https://pages.nist.gov/OSCAL/",
```
Comment (david-waltermire) [inline src/main/kotlin/gov/nist/secauto/oscal/tools/server/OscalVerticle.kt:454]:
```suggestion
                  "informationUri": "https://github.com/metaschema-framework/oscal-server",
```

**Comment by david-waltermire** [comment]
This is an incremental step towards providing REST-based capabilities around metaschema-java based operations.

When should we consider a more formal REST API that directly supports these operations without using the CLI? For example, a REST API that directly uses the underlying metaschema-java core, databind, and schemagen APIs?

**Comment by wandmagic** [comment]
> This is an incremental step towards providing REST-based capabilities around metaschema-java based operations.
> 
> 
> 
> When should we consider a more formal REST API that directly supports these operations without using the CLI? For example, a REST API that directly uses the underlying metaschema-java core, databind, and schemagen APIs?

I believe if we were to expand the scope of the server for generic storage and registration of metaschema models, we would certainly get strong benefit using the api directly, currently there are some drawbacks to taking this approach ie log4j logging must be at info level for cli to work and we must write output to a file and then read it back. However for our current use case of making our constraint test suite run faster, having parity with the cli I think is a benefit. To answer your question directly, when we should use the api directly instead of doing commands? I think when we intend to leverage the service in a way that goes beyond the cli or when we want to tighten performance and code quality so that the server can be embedded in a native application or private cloud.

**Review by aj-stein-gsa — APPROVED** [review]

**Pull request merged**